--- bin/nvcc.profile.bak	2024-03-12 15:52:23.511145665 +0000
+++ bin/nvcc.profile	2024-03-12 16:27:28.806624738 +0000
@@ -1,16 +1,16 @@
 
-TOP              = $(_HERE_)/..
+TOP              = $(_HERE_)/../$(_TARGET_DIR_)
 
 CICC_PATH        = $(TOP)/nvvm/bin
 CICC_NEXT_PATH   = $(TOP)/nvvm-next/bin
 NVVMIR_LIBRARY_DIR = $(TOP)/nvvm/libdevice
 
 LD_LIBRARY_PATH += $(TOP)/lib:
-PATH            += $(CICC_PATH):$(_HERE_):
+PATH            += $(TOP)/bin:$(CICC_PATH):$(TOP)/../../bin:$(TOP)/../../$(_NVVM_BRANCH_)/bin:
 
-INCLUDES        +=  "-I$(TOP)/$(_TARGET_DIR_)/include" $(_SPACE_)
+INCLUDES        +=  "-I$(TOP)/include" $(_SPACE_)
 
-LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)/stubs" "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)"
+LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/lib/stubs" "-L$(TOP)/lib"
 
 CUDAFE_FLAGS    +=
 PTXAS_FLAGS     +=

