--- bin/nvcc.profile.bak	2025-05-03 14:44:20
+++ bin/nvcc.profile	2025-05-05 09:14:39
@@ -1,16 +1,16 @@

-TOP              = $(_HERE_)/..
+TOP              = $(_HERE_)/../$(_TARGET_DIR_)

 CICC_PATH        = $(TOP)/nvvm/bin
 NVVMIR_LIBRARY_DIR = $(TOP)/nvvm/libdevice

 LD_LIBRARY_PATH += $(TOP)/lib:
-PATH            += $(CICC_PATH):$(_HERE_):
+PATH            += $(TOP)/bin:$(CICC_PATH):$(TOP)/../../bin:$(TOP)/../../$(_NVVM_BRANCH_)/bin:

-INCLUDES        +=  "-I$(TOP)/$(_TARGET_DIR_)/include" $(_SPACE_)
-SYSTEM_INCLUDES +=  "-isystem" "$(TOP)/$(_TARGET_DIR_)/include/cccl" $(_SPACE_)
+INCLUDES        +=  "-I$(TOP)/include" $(_SPACE_)
+SYSTEM_INCLUDES +=  "-isystem" "$(TOP)/include/cccl" $(_SPACE_)

-LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)/stubs" "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)"
+LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/lib/stubs" "-L$(TOP)/lib"

 CUDAFE_FLAGS    +=
 PTXAS_FLAGS     +=

